src/qt/veil/forms/settingsfaq12.ui

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>SettingsFaq12</class>
+ <widget class="QWidget" name="SettingsFaq12">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>656</width>
+    <height>575</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <property name="styleSheet">
+   <string notr="true">font-size:16px;
+color:#707070;</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="rightMargin">
+    <number>60</number>
+   </property>
+   <item>
+    <widget class="QLabel" name="label_2">
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>0</height>
+      </size>
+     </property>
+     <property name="styleSheet">
+      <string notr="true">font-weight:200;</string>
+     </property>
+     <property name="text">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Unlocking/Locking the wallet from within the GUI&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;At the bottom right of the veil wallet GUI a padlock icon is displayed showing the current lock status of your wallet. Clicking the padlock when it shows the locked state will display a prompt asking you for your wallet password. If entered correctly, your wallet will be unlocked.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;To lock your wallet again you click the now unlocked padlock.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Editor changes:
Unlocking/locking the wallet from within the GUI
At the bottom right of the Veil wallet GUI a padlock icon is displayed showing the current lock status of your wallet. 
Clicking the padlock when it shows the locked state will display a prompt asking you for your wallet password.
If entered correctly, your wallet will be unlocked.
To lock your wallet again click the now unlocked padlock icon.

`&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Unlocking/locking the wallet from within the GUI&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;At the bottom right of the Veil wallet GUI a padlock icon is displayed showing the current lock status of your wallet. Clicking the padlock when it shows the locked state will display a prompt asking you for your wallet password. If entered correctly, your wallet will be unlocked.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;To lock your wallet again click the now unlocked padlock icon.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;`
^^^ This has the changes applied. Copy, paste, done.